README.md

```diff
@@ -7,7 +7,7 @@ You can install using:
 ```
 curl -s https://getcomposer.org/installer | php
 php composer.phar install
-composer require zendframework/zendservice-recaptcha
+php composer.phar require zendframework/zendservice-recaptcha
 ```
```

Comment:
Alternately, we point them to the Composer download page, and demonstrate using `composer require` (not usage of the `phar` file). Either way works, though we've been trending towards just saying `composer require`.